src/components/subscription/ExtendSubscriptionModal.tsx

```
@@ -0,0 +1,174 @@
+            Extend Subscription
+          </DialogTitle>
+          <DialogDescription className="text-white/60 font-light">
+            Add months to your subscription expiry date. You can pay multiple months in advance.
+          </DialogDescription>
+        </DialogHeader>
+        
+        <div className="py-4">
+          <div className="mb-4">
+            <label className="block text-sm font-light text-white/70 mb-2">
+              Number of months
+            </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)